packages/documentation/src/stories/components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,305 @@
+import React from 'react';
+import { Meta, Story, Args } from "@storybook/react";
+import { useArgs } from '@storybook/client-api';
+import parse from 'html-react-parser';
+import docsPage from './alert.docs.mdx';
+
+export default {
+  title: 'Components/Alert',
+  parameters: {
+    docs: {
+      page: docsPage
+    },
+    controls: {
+      exclude: [
+        'Title',
+        'Content'
+      ]
+    }
+  },
+  args: {
+    title: 'Titulum',
+    content: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
+    variant: 'alert-primary',
+    noIcon: false,
+    icon: 'null',
+    dismissible: false,
+    fixed: false,
+    action: false,
+    show: true
+  },
+  argTypes: {
+    title: {
+      name: 'Title',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    content: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: { category: 'Content' }
+    },
+    variant: {
+      name: 'Variant',
+      description: 'Defines a style variant.',
+      control: {
+        type: 'radio',
+        labels: {
+          'alert-primary': 'Primary',
+          'alert-success': 'Success',
+          'alert-danger': 'Danger',
+          'alert-warning': 'Warning',
+          'alert-info': 'Info'
+        }
+      },
+      options: [
+        'alert-primary',
+        'alert-success',
+        'alert-danger',
+        'alert-warning',
+        'alert-info'
+      ],
+      table: {
+        category: 'General'
+      }
+    },
+    noIcon: {
+      name: 'No Icon',
+      description: 'Removes the predefined icon completely.',
+      control: {
+        type: 'boolean'
+      },
+      table: {
+        category: 'General'
+      }
+    },
+    icon: {
+      name: 'Icon',
+      description: 'Defines a custom icon.',
+      if: {
+        arg: 'noIcon',
+        truthy: false
+      },
+      control: {
+        type: 'select',
+        labels: {
+          'null': 'Default',
+          '1001': 'Envelope (1001)',
+          '2023': 'Cog (2023)',
+          '2025': 'Send (2025)',
+          '2035': 'Home (2035)',
+          '2101': 'Bubble (2101)'
+        }
+      },
+      options: [
+        'null',
+        '1001',
+        '2023',
+        '2025',
+        '2035',
+        '2101'
+      ],
+      table: {
+        category: 'General'
+      }
+    },
+    dismissible: {
+      name: 'Dismissible',
+      description: 'Adds the dismissible styles.<br/><div className="alert alert-info" style="padding: 0.2rem 0.5rem 0.2rem 2rem; min-height: 0; background-position: 0.5rem center; background-size: 1rem; font-size: inherit;">Do not forget to add the structural adjustments!</div>',
```

Comment:
Descriptions are HTML strings, so they should use `class`, not `className`.

Also, they are displayed within spans in the args table, so it's best to only use inline elements and not divs.

```suggestion
      description: 'Adds the dismissible styles.<br/><span class="bg-info rounded light lh-1 d-flex align-items-center gap-mini py-micro px-mini mt-mini"><em class="pi pi-2106 pi-small"></em>Do not forget to add the structural adjustments!</span>',
```

Reply:
Hei @alizedebray 
Thanks for your latest review!

- The "unwanted padding around the alerts header" is coming from the file `@use './layouts/portal';` inside of the `@swisspost/design-system-styles/index.scss`. I don't know why this styles are not present in the generated `@swisspost/design-system-styles/index.css`, but it has nothing to the with a wrong setup of the alert component.
- Descriptions are html strings, that get rendered by the storybook react app and therefore, they need to have the attribute `className` instead of `class`. In the output they are renamed to `class="..."` for sure.
- Your right! But if we do so, we could break some of the expected behaviour on our own components (for example, that the alert should be display: block; per default, etc.).
Because of this reason, I would see this as a downside of storybook, which we can not change, but leave our code as it shoud be.
Is this ok for you?